tools/sdk-cli/src/command_handlers.rs

```diff
@@ -215,11 +215,30 @@ pub(crate) async fn handle_command(
             include_failures,
             limit,
             offset,
+            metadata_filters: metadata_filters_raw,
         } => {
+            let metadata_filters = match metadata_filters_raw {
+                Some(raw_filters) => {
+                    let mut filters = vec![];
+
+                    for filter in raw_filters.iter() {
+                        let i = filter.find(':').context("Invalid metadata filter")?;
+
+                        filters.push(MetadataFilter {
+                            json_path: filter[0..i].to_string(),
+                            json_value: filter[i + 1..].to_string(),
+                        });
```

Comment:
Perhaps the more proper solution @dangeross. Let me know what you think

Reply:
Or maybe `filter.split_once(':')`?

Reply:
Guess that's just more Rust magic to know of. Fixed.